Lib/test/test_sys_settrace.py

```diff
@@ -1167,21 +1175,71 @@ def test_no_jump_into_finally_block(output):
             output.append(6)
         output.append(7)
 
-    @jump_test(1, 5, [], (ValueError, 'finally'))
-    def test_no_jump_into_finally_block_2(output):
+    @jump_test(5, 1, [1, 3], (ValueError, "out of a 'finally'"))
+    def test_no_jump_out_of_finally_block(output):
         output.append(1)
         try:
             output.append(3)
         finally:
             output.append(5)
 
-    @jump_test(5, 1, [1, 3], (ValueError, 'finally'))
-    def test_no_jump_out_of_finally_block(output):
+    @jump_test(1, 5, [], (ValueError, "into an 'except'"))
+    def test_no_jump_into_bare_except_block(output):
         output.append(1)
         try:
             output.append(3)
-        finally:
+        except:
+            output.append(5)
+
+    @jump_test(1, 5, [], (ValueError, "into an 'except'"))
+    def test_no_jump_into_qualified_except_block(output):
+        output.append(1)
+        try:
+            output.append(3)
+        except Exception:
+            output.append(5)
+
+    @jump_test(3, 6, [2, 5, 6], (ValueError, "into an 'except'"))
+    def test_no_jump_into_bare_except_block_from_try_block(output):
+        try:
+            output.append(2)
+            output.append(3)
+        except:  # still executed if the jump is failed
```

Comment:
Using 'still' in the comment is a little bit misleading, the except clause is executed because the jump is failed otherwise it would not be executed.